Create interests_page.tsx

middleton258 opened the pull request:
Created interest page for selecting interests for clubs; buttons takes users to tinder swipe

middleton258 committed:
Create interests_page.tsx

Created interest page for selecting interests for clubs; buttons takes users to  tinderswipe

ldfMion reviewed (changes requested):
- Missing ShadCn components
- Not using Next.js Link component for navigation
- Not using Tailwind
- The component isn't used anywhere, how did you even test?

middleton258 committed:
Requested Updates to Interests Page

Renamed interests_page.tsx to page.tsx; updated to Link tag for navigation; switched to using cn components; switched to tailwind for styling

Added Link to layout.tsx to add match button to header

ldfMion committed:
Merge branch 'dev' into nolanm/interest-section

ldfMion committed:
integrate into release
